profiles/commands.py

```diff
@@ -13,11 +13,22 @@
 
 
 def update_profile_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
-    LOGGER.info('Updating profile %s with ORCID record %s', profile.id, orcid_record.get('path'))
-    _update_name_from_orcid_record(profile, orcid_record)
-    _update_affiliations_from_orcid_record(profile, orcid_record)
-    _update_email_addresses_from_orcid_record(profile, orcid_record)
-
+    try:
+        LOGGER.info('Updating profile %s with ORCID record %s', profile.id, orcid_record.get('path'))
+        _update_name_from_orcid_record(profile, orcid_record)
+        _update_affiliations_from_orcid_record(profile, orcid_record)
+        _update_email_addresses_from_orcid_record(profile, orcid_record)
+    except:
```

Comment:
other exceptions are being thrown, like ProfileNotFound that we don't care about.